### ISSUE TITLE
Downloading attachment result in file called "content" instead of listed filename in email

### ISSUE THREAD

**Issue by SchietSchijf** [rnwood/smtp4dev#113]
Hi,

I want to download an attachment (pdf in this case). If I click the download button, it will save the file as "content" without extention, while a proper filename with extention is provided.

Used build: Smtp4dev-win-x64-v3.1.0-ci0448

snippet of the raw message:

> `Content-Type: application/octet-stream; name="Achzet, , Jeannette_364524.pdf"
Content-Transfer-Encoding: base64
Content-Disposition: attachment

JVBERi0xLjUNCiW1tbW1DQoxIDAgb2JqDQo8PC9UeXBlL0NhdGFsb2cvUGFnZXMgMiAw<...>`

**Comment by SchietSchijf** [comment]
As far as I can see, the issue is not fixed with the new release (Smtp4dev-win-x64-v3.1.0-ci0470)

The attachment is still downloaded as "content". I tested this in Chrome and Firefox, same result.

I have attached the log of this email.
[message log.txt](https://github.com/rnwood/smtp4dev/files/2992059/message.log.txt)

**Comment by SchietSchijf** [comment]
Thanks for the quick fixes but I still get the filename "content", even with other attachments
log of a txt attachment: 
[message log2.txt](https://github.com/rnwood/smtp4dev/files/2992477/message.log2.txt)

**Comment by rnwood** [comment]
Hi

Thanks for your example message. Sorry I didn't mean my initial commit to close this issue.

I'm not able to reproduce this with build 472 with your example message log2.txt:
![image](https://user-images.githubusercontent.com/1327895/54761501-1f3c8080-4bea-11e9-8d87-e5035e32ddbc.png)

**Comment by SchietSchijf** [comment]
In firefox I get this: ![image](https://user-images.githubusercontent.com/4268326/54762158-d4c00180-4bf3-11e9-9690-51125f8be585.png)
Even displaying the email different. Chrome is displaying it the same as in my firefox screenshot
I'm running smtp4dev on Windows 7

**Comment by SchietSchijf** [comment]
Anything I can do to give you more information?

**Comment by rnwood** [comment]
Can you confirm that last screenshot is when running build 472? The version is printed on the first line when running the .exe.

**Comment by SchietSchijf** [comment]
The issue has been resolved by the previous commit.
I didn't see the fix because I made a shortcut on one of the older builds to have it listen on all interfaces but it still referred to the older build.

Now I get the same view as you have: 
![image](https://user-images.githubusercontent.com/4268326/54780627-85d99280-4c1a-11e9-9547-f32a96366778.png)


Sorry for the trouble and thank you for fixing it!
